### ISSUE TITLE
cover security

### ISSUE THREAD

**Issue by aladdin-add** [survivejs/maintenance-book#99]
some thoughts(but it should be more):
+ it is recommended to enable 2fa.
+ `npm audit`

**Comment by sapegin** [comment]
I've just added a few bits on `npm audit` in #97. Would you add anything else?

Do you mean 2FA on npm, GitHub or 2FA in general?

**Comment by aladdin-add** [comment]
> Do you mean 2FA on npm, GitHub or 2FA in general?

2FA in general, but npm seems to be more important.(since its have more downloads).
The idea is initially inspired by https://eslint.org/blog/2018/07/postmortem-for-malicious-package-publishes

**Comment by sapegin** [comment]
Yeah, that's a tricky thing because it doesn't work with tools like semantic-release. Worth mentioning that.
